inyoka_theme_default/templates/wiki/tag_cloud.html

```diff
@@ -0,0 +1,41 @@
+{#
+    wiki/tag_cloud.html
+    ~~~~~~~~~~~~~~~~~~~
+
+    This template is used to show a tag cloud of the most used wiki tags.
+
+    :copyright: (c) 2007-2016 by the Inyoka Team, see AUTHORS for more details.
```

Comment:
We should talk about the copyright begin of the files from the new theme, since it is inconsistent.